SocialFilter/sections/specific.txt

```diff
@@ -47,6 +47,7 @@ originalnews.nico,archives.gov,sportnews.to,nationalchronicle.in,analogictips.co
 sedaily.com,canshop.jp,koreatimes.co.kr,m.chosun.com,sports.donga.com##.sns_area
 !
 dekiru.net##.module-individual-article-item-share
+kakakumag.com##.article-sns-btm
```

Comment:
There is the same block at the top - so it's better to use `kakakumag.com##div[class^="article-sns-"]`

Reply:
ok